### ISSUE TITLE
27056568: VoiceOver does not read the `alt` text of `<img>` elements with an SVG source

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#15197]
#### Description

When using VoiceOver, the alt text of `<img>` elements with an SVG src are not read. This includes `<img>` elements inside `<picture>` elements.

Steps to Reproduce:
1. With VoiceOver enabled navigate to https://dorigo.co/svg-test/
2. Place finger on first logo — this is an `<img>` inside a `<picture>`. The source used should be the SVG.
3. Place finger on second logo — this is an `<img>` with an SVG as the source.
4. Place finder on third logo — this is an `<img>` with a PNG as the source.

Expected Results:
For all of the images, VoiceOver should read the alt and title text, and that the user is touching an image.

Actual Results:
VoiceOver plays a “ding” noise when touching the first two images. For the third image, with a PNG source, the alt and title text are read.

Notes:
This issue also affects Safari and Safari Technology Preview on macOS.
## 

Product Version: 9.3.2 (13F69)
Created: 2016-06-28T16:25:09.828690
Originated: 2016-06-28T17:23:00
Open Radar Link: http://www.openradar.me/27056568
